textinput_model.go

```diff
@@ -30,114 +30,131 @@ func (p *textinputPlugin) addChar(char []rune) {
 
 	p.selectionBase += len(char)
 	p.selectionExtent = p.selectionBase
-	p.updateEditingState()
```

Comment:
If you remove this statement, the dart textinput will be update on the next key `glfwKeyCallback`, the dart and go aren't in sync.
Text can be added with `glfwCharCallback` add `p.updateEditingState()` at the end of the method handler.